places-solution-integrations/campaign/postbacks.md

```diff
@@ -0,0 +1,76 @@
+# Sending Places Data to Campaign via Launch Rules
+
+This document assumes that you have Adobe Places implemented in your application. If you need help implementing Adobe Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).
```

Comment:
Change to:
"The following information assumes that you have implemented Places in your application. For more information about  implementing Places, see [Places Extensions for AEP SDK](../../configure-places-in-the-sdk/README.md).